src/main/java/com/epam/config/DBConfiguration.java

```diff
@@ -28,4 +29,9 @@ public DataSource dataSource() {
     public NamedParameterJdbcTemplate namedParamJdbcTemplate() {
         return new NamedParameterJdbcTemplate(dataSource());
     }
+
+    @Bean
+    public JdbcTemplate simpleJdbcTemplate() {
```

Comment:
Hmm, but we already have `NamedParametrJdbcTemplate`. Please use only one of them in the project

Reply:
If i'll drop NamedParametJdbcTemplate, i'll need to change classes which out of my task, like "Person".... Can i do it? Or it must be another little pull request for fix?

Reply:
But why u cant use NamedParametJdbcTemplate instead of JdbcTemplate?

Reply:
Because this type of template is old. Last versions of JdbcTemplate support all feauters which contains in NamedParametJdbcTemplate.

Reply:
Okay, then create a task in Trello to migrate from NamedParametJdbcTemplate to JdbcTemplate